src/App.js

```diff
@@ -1,7 +1,15 @@
-import "App.css";
 import { Outlet } from "react-router-dom";
+import ReactGA from 'react-ga';
+
+import "App.css";
+
+ReactGA.initialize(process.env.REACT_APP_GA_TRACKING_ID);
```

Comment:
I have added the env variable with this key in vercel. So when the build passes, the analytics will work.